target/linux/mediatek/dts/mt7981b-nradio-c8-668gl.dts

```diff
@@ -0,0 +1,264 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "mt7981.dtsi"
+
+/ {
+	model = "NRadio C8-668GL";
+	compatible = "nradio,c8-668gl", "mediatek,mt7981";
+
+	aliases {
+		ethernet0 = &gmac0;
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+		serial0 = &uart0;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n1";
+		/* If official system version < 1.9.2.n6.c3, use root=PARTLABEL=rootfs */
+		bootargs = "console=ttyS0,115200n1 root=PARTLABEL=rootfs_2nd rootwait";
+	};
+
+	memory {
+		reg = <0 0x40000000 0 0x40000000>;
+	};
+
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&pio 1 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&pio 9 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		cpepower {
+			gpio-export,name = "cpe-pwr";
+			gpio-export,output = <0>;
+			gpios = <&pio 31 GPIO_ACTIVE_LOW>;
+		};
+
+		cpesel0 {
+			gpio-export,name = "cpe-sel0";
+			gpio-export,output = <1>;
+			gpios = <&pio 30 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_status: status {
+			label = "hc:blue:status";
+			gpios = <&pio 10 GPIO_ACTIVE_LOW>;
+		};
+
+		cmode5 {
+			label = "hc:blue:cmode5";
+			gpios = <&pio 11 GPIO_ACTIVE_LOW>;
+		};
+
+		cmode4 {
+			label = "hc:blue:cmode4";
+			gpios = <&pio 12 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi {
+			label = "hc:blue:wifi";
+			gpios = <&pio 13 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Please use new format. An example: https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/dts/mt7986a-ruijie-rg-x60-pro.dts#L49

Reply:
> Please use new format. An example: https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/dts/mt7986a-ruijie-rg-x60-pro.dts#L49

Thank you for your suggestion! I’ve updated the format as per the example you provided. The updated content is as follows:

```dts
leds {
	compatible = "gpio-leds";

	led_power: power {
		color = <LED_COLOR_ID_BLUE>;
		function = LED_FUNCTION_POWER;
		gpios = <&pio 10 GPIO_ACTIVE_LOW>;
	};

	led-cellular-5g {
		color = <LED_COLOR_ID_BLUE>;
		function = LED_FUNCTION_INDICATOR;
		function-enumerator = <0>;
		gpios = <&pio 11 GPIO_ACTIVE_LOW>;
	};

	led-cellular-4g {
		color = <LED_COLOR_ID_BLUE>;
		function = LED_FUNCTION_INDICATOR;
		function-enumerator = <1>;
		gpios = <&pio 12 GPIO_ACTIVE_LOW>;
	};

	led-wifi {
		color = <LED_COLOR_ID_BLUE>;
		function = LED_FUNCTION_WLAN;
		gpios = <&pio 13 GPIO_ACTIVE_LOW>;
	};
};
```